src/metabase/lib/drill_thru/column_extract.cljc

```diff
@@ -45,11 +45,12 @@
              (nil? value)
              (lib.drill-thru.common/mbql-stage? query stage-number))
     (when-let [drill (column-extract-drill-for-column query column)]
-      (merge drill
-             {:lib/type :metabase.lib.drill-thru/drill-thru
-              :type     :drill-thru/column-extract}
-             (lib.drill-thru.column-filter/prepare-query-for-drill-addition
-               query stage-number column column-ref :expression)))))
+      (when-let [drill-details (lib.drill-thru.column-filter/prepare-query-for-drill-addition
+                                 query stage-number column column-ref :expression)]
+        (merge drill
+               drill-details
+               {:lib/type :metabase.lib.drill-thru/drill-thru
+                :type     :drill-thru/column-extract})))))
```

Comment:
Is the change in the order of the arguments of `merge` intentional?

Reply:
They should be harmless. There are no common properties